optaplanner-examples/src/test/java/org/optaplanner/examples/vehiclerouting/optional/score/VehicleRoutingConstraintProviderTest.java

```diff
@@ -30,8 +30,8 @@
 
 public class VehicleRoutingConstraintProviderTest {
 
-    private final ConstraintVerifier<VehicleRoutingConstraintProvider, VehicleRoutingSolution> constraintVerifier
-            = ConstraintVerifier.build(new VehicleRoutingConstraintProvider(),
+    private final ConstraintVerifier<VehicleRoutingConstraintProvider, VehicleRoutingSolution> constraintVerifier = ConstraintVerifier
```

Comment:
Impossible to break line around assignment operator -> forces to violate line length limit.